docs/CHANGELOG.asciidoc

```diff
@@ -8,16 +8,2915 @@
 :issue: https://github.com/elastic/kibana/issues/
 :pull: https://github.com/elastic/kibana/pull/
 
-Review important information about the {kib} 8.0.0 releases.
+Review important information about the {kib} 8.6.0 release.
+
+* <<release-notes-8.6.0>>
+//* <<release-notes-8.5.2>>
+//* <<release-notes-8.5.1>>
+//* <<release-notes-8.5.0>>
+//* <<release-notes-8.4.3>>
+//* <<release-notes-8.4.2>>
+//* <<release-notes-8.4.1>>
+//* <<release-notes-8.4.0>>
+//* <<release-notes-8.3.3>>
+//* <<release-notes-8.3.2>>
+//* <<release-notes-8.3.1>>
+//* <<release-notes-8.3.0>>
+//* <<release-notes-8.2.3>>
+//* <<release-notes-8.2.2>>
+//* <<release-notes-8.2.1>>
+//* <<release-notes-8.2.0>>
+//* <<release-notes-8.1.3>>
+//* <<release-notes-8.1.2>>
+//* <<release-notes-8.1.1>>
+//* <<release-notes-8.1.0>>
+//* <<release-notes-8.0.1>>
+//* <<release-notes-8.0.0>>
+//* <<release-notes-8.0.0-rc2>>
+//* <<release-notes-8.0.0-rc1>>
+//* <<release-notes-8.0.0-beta1>>
+//* <<release-notes-8.0.0-alpha2>>
+//* <<release-notes-8.0.0-alpha1>>
 
-* <<release-notes-8.0.0>>
-* <<release-notes-8.0.0-rc2>>
-* <<release-notes-8.0.0-rc1>>
-* <<release-notes-8.0.0-beta1>>
-* <<release-notes-8.0.0-alpha2>>
-* <<release-notes-8.0.0-alpha1>>
+--
+
+[[release-notes-8.6.0]]
+== {kib} 8.6.0
+
+coming::[8.6.0]
+
+Review the following information about the {kib} 8.6.0 release.
+
```

Comment:
```suggestion
Review the following information about the {kib} 8.6.0 release.

[float]
[[known-issues-8.6.0]]
=== Known issues

[discrete]
[[known-issue-146020]]
.Attempting to create APM latency threshold rules from the Observability rules page fail
[%collapsible]
====
*Details* +
When you attempt to create an APM latency threshold rule in **Observability** > **Alerts** > **Rules** for all services or all transaction types, the request will fail with a `params invalid` error.
*Impact* +
This known issue only impacts the Observability Rules page. To work around this issue, create APM latency threshold rules in the APM Alerts and Rules dialog. See {kibana-ref}/apm-alerts.html[Alerts and rules] for more information.
====

```

Reply:
cc @cauemarcondes and @KOTungseth for content and wording suggestions

Reply:
@bmorelli25 LGTM